### PR TITLE
C4

### PR DESCRIPTION
**Pull request by anburocky3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Sample project where we will learn the basic fundamentals of git versioning syst
 ## Changlogs:
 
 1. Made some changes
+2. Changed one.txt
+3. Commit 3
```
Comment (void0err) [inline README.md:9]: is this commit necessary?

**Review by void0err — COMMENTED** [review]
Simple commetn

**Review by void0err — CHANGES_REQUESTED** [review]
Change some files

File: README.md (hunk 1 above)
Comment (void0err) [inline README.md:9]: Do we need this commit?

Comment (void0err) [inline README.md:9]: Do we need this commit?

Comment (anburocky3) [reply README.md:9]: Yes, we are testing this commit

**Review by void0err — APPROVED** [review]
Approving this pull request

**Review by void0err — APPROVED** [review]
Approving this pull request

**Pull request merged**